bitnami/grafana-loki/values.yaml

```diff
@@ -4462,455 +4462,175 @@ tableManager:
     ##
     ingressNSMatchLabels: {}
     ingressNSPodMatchLabels: {}
-## @section Promtail Deployment Parameters
+## @section Grafana Alloy Deployment Parameters
 ##
-promtail:
-  ## @param promtail.enabled Deploy promtail
+grafanaalloy:
```

Comment:
You're still overriding tooooooo many sub-chart values. Check Thanos (or any other chart using sub-charts) and the MinIO values that it overrides as an example:
- https://github.com/bitnami/charts/blob/main/bitnami/thanos/values.yaml#L5070-L5113

Reply:
Reducing them